File: src/main/java/hu/telekom/tkriszti/flightInfo/dto/ResultDTO.java
package hu.telekom.tkriszti.flightInfo.dto;

import hu.telekom.tkriszti.flightInfo.model.Pilot;

import java.util.List;

public final class ResultDTO {
	
	private final String pilotName;
	private final int pilotLicenceYear;
	private final List<Integer> flightsByPilot;
	private final int totalFlightTime;

	public ResultDTO(String pilotName, int pilotLicenceYear, List<Integer> flightsByPilot, int totalFlightTime) {
		this.pilotName = pilotName;
		this.pilotLicenceYear = pilotLicenceYear;
		this.flightsByPilot = flightsByPilot;
		this.totalFlightTime = totalFlightTime;
	}

	public String getPilotName() {
		return pilotName;
	}

	public int getPilotLicenceYear() {
		return pilotLicenceYear;
	}

	public List<Integer> getFlightsByPilot() {
		return flightsByPilot;
	}

	public int getTotalFlightTime() {
		return totalFlightTime;
	}

	@Override
	public String toString() {
		return String.format(pilotName + " " + pilotLicenceYear +"%n%n" + flightsByPilot +  totalFlightTime );
	}
}

File: src/main/java/hu/telekom/tkriszti/flightInfo/dto/ResultDTO.kt
package hu.telekom.tkriszti.flightInfo.dto

class ResultDTO (
    private var pilotName: String? = null,
    private var pilotLicenceYear: Int = 0,
    private var flightsByPilot: List<Int>? = null,
    private var totalFlightTime: Int = 0 ) {
    fun getPilotName(): String? {
        return pilotName
    }

    fun getPilotLicenceYear(): Int {
        return pilotLicenceYear
    }

    fun getFlightsByPilot(): List<Int>? {
        return flightsByPilot
    }

    fun getTotalFlightTime(): Int {
        return totalFlightTime
    }

    override fun toString(): String {
        return String.format("$pilotName $pilotLicenceYear Flights: $flightsByPilot $totalFlightTime minutes")
    }
}